src/api/app/policies/bs_request_policy.rb

```diff
@@ -7,4 +7,10 @@ def create?
 
     false
   end
+
+  def handle_request?
```

Comment:
This is not a policy. You are not deciding if someone can or can't do something.

Reply:
While it might be a bit stretching the definition of a policy, the alternative is to put this check in the controller and pass it around through multiple view levels. I prefer this approach with the policy.

Reply:
You can always push this to the model

Reply:
I liked the policy approach because it is self-contained and self-computed, no parameters to pass, no variables to carry on, etc.
I also agree it is not 100% a policy, it tells if someone can or cannot do something but it does not prevent anything in the end other than rendering a message.
Moving to the model could work but it does not really belong to the object model to be honest. I'm fine with the policy approach, but I don't want to enforce it.
I could see as an option to move it to the `user_helper.rb` class, because it is a user property more than a request one. The drawback still is to pass parameters around.
Opinions?

Reply:
I'm fine with having this in the policy